sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/util/ChunkedDownloadUtils.java

```diff
@@ -32,10 +34,11 @@ public class ChunkedDownloadUtils {
     Download the first chunk. Construct a Mono which will emit the total count for calculating the number of chunks,
     access conditions containing the etag to lock on, and the response from downloading the first chunk.
      */
+    @SuppressWarnings("unchecked")
```

Comment:
What does this suppression do?

Reply:
There is no easy way to resolve the warning programmatically - and there are other instances in the SDK where Optional<T>.get is used with @SuppressWarnings("unchecked")